### ISSUE TITLE
Big pose visualization is strange

### ISSUE THREAD

**Issue by Trainingzy** [skhu101/GauHuman#22]
Thanks a lot for your great work!

I visualize the initialization of gaussians ([big pose xyz](https://github.com/skhu101/GauHuman/blob/85b2bfdf9091ca7c36d372ea2467f1e0b933f01b/scene/dataset_readers.py#L342)). I save these point clouds and view them in meshlab.

The result is as this, all points are spread out. I think it should be dense point clouds like the canonical space gaussian in Fig 2 of the paper.
<img width="580" alt="image" src="https://github.com/skhu101/GauHuman/assets/88388847/405bf85b-e4fa-475c-aa50-29783a87faad">

**Comment by skhu101** [comment]
Hi, thanks for your question. It would be quite strange as the initialization of Gaussians should be exactly the SMPL vertices. Could you show me the code for saving big_pose_xyz?

**Comment by Trainingzy** [comment]
I just use StorePly provided by your code.

I think I found the solution. I tried on another server and it shows correct results. In my previous trial, there seems to be a precision issue. I convert the arrays to float32 and fix this bug.

For those facing the same problem, consider convert the variables to np.float32.

![image](https://github.com/skhu101/GauHuman/assets/88388847/31544eef-e376-4a99-96c2-52a73e6dc0b0)
